src/issues/issue-2/fav-smells/index.js

```diff
@@ -153,6 +155,42 @@ const ControlArrows = styled.button`
       z-index: 10;
     `}
 `
+const Dot = styled.div`
```

Comment:
since `Dot` and `SelectedDot` are almost identical, I would just make one `Dot` and modify the style via props